src/interfaces.ts

```diff
@@ -56,7 +56,11 @@ export enum ConfigSettings {
     OTP_HASHING_SECRET = "OTP_HASHING_SECRET",
     OTP_HASHING_SALT = "OTP_HASHING_SALT",
     OTP_EXPIRY_TIME = "OTP_EXPIRY_TIME",
-    MESSAGEBIRD_API_KEY = "MESSAGEBIRD_API_KEY"
+    MESSAGEBIRD_API_KEY = "MESSAGEBIRD_API_KEY",
+    HTTPS_PROXY_HOST = "HTTPS_PROXY_HOST",
+    HTTPS_PROXY_PORT = "HTTPS_PROXY_PORT",
+    HTTPS_PROXY_USERNAME = "HTTPS_PROXY_USERNAME",
+    HTTPS_PROXY_PASSWORD = "HTTPS_PROXY_PASSWORD"
```

Comment:
Not to nit pick, but lets keep them in order?
```suggestion
    HTTPS_PROXY_PASSWORD = "HTTPS_PROXY_PASSWORD"
    HTTPS_PROXY_USERNAME = "HTTPS_PROXY_USERNAME",
```